specification/recoveryservicessiterecovery/resource-manager/readme.typescript.md

```diff
@@ -7,6 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-recoveryservices-siterecovery"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-recoveryservices-siterecovery"
+  output-folder: "$(typescript-sdks-folder)/sdk/recoveryservices-siterecovery/arm-recoveryservices-siterecovery"
```

Comment:
Should this be "recoveryservicessiterecovery" without a "-"?

Reply:
yes